### PR TITLE
cut 2.4 branch for ocm curator

### PR DESCRIPTION
**Pull request by elgnay**
The related issue:https://github.com/open-cluster-management/backlog/issues/14512 
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/cluster-curator-controller/open-cluster-management-cluster-curator-controller-release-2.4-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    - release-2.4
+    cluster: build01
+    context: ci/prow/images
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-open-cluster-management-cluster-curator-controller-release-4-images
```
Comment (zhiweiyin318) [inline ci-operator/jobs/open-cluster-management/cluster-curator-controller/open-cluster-management-cluster-curator-controller-release-2.4-presubmits.yaml:15]: 2.4 ?

Comment (elgnay) [reply ci-operator/jobs/open-cluster-management/cluster-curator-controller/open-cluster-management-cluster-curator-controller-release-2.4-presubmits.yaml]: updated

**Pull request merged**